rslib/src/search/parser.rs

```diff
@@ -118,6 +118,13 @@ pub enum TemplateKind<'a> {
     Name(Cow<'a, str>),
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub enum EaseKind {
+    Rated(u8),
+    Reviewed,
+    Manually,
```

Comment:
What do you think about changing the wording to the following?

AnswerButton(u8)
AnyAnswerButton
ManualReschedule

Reply:
That's probably clearer, will fix it.